ui/analyse/src/study/practice/studyPracticeCtrl.ts

```diff
@@ -60,7 +60,7 @@ export default function (root: AnalyseCtrl, studyData: StudyData, data: StudyPra
   function onVictory(): void {
     saveNbMoves();
     lichess.sound.play('practiceSuccess');
-    if (autoNext()) setTimeout(goToNext, 1000);
+    if (studyData.chapter.practice) if (autoNext()) setTimeout(goToNext, 1000);
```

Comment:
This should use `&& instead of a doubel `if, i.e. `studyData.chapter.practice && autoNext()`